helm/vitess/templates/_vttablet.tpl

```diff
@@ -240,6 +240,12 @@ spec:
               fi
               sleep 5
             done
+{{ if $keyspace.schema }}
```

Comment:
I think this probably needs to be implemented as another job, or maybe tacked onto the vschema job, since this will only run once. If ISM has already run, this job exits early, but people will still assume that changes to their schemas are being applied.

Reply:
Good idea. I've now created a _shard.tpl, and also moved ISM there. Let me know what you think.